ansible/playbooks/roles/kubernetes_node/templates/kubeadm-join-node.yml.j2

```diff
@@ -10,3 +10,4 @@ nodeRegistration:
   kubeletExtraArgs:
     enable-controller-attach-detach: "false"
     node-labels: {{ specification.node_labels }}
+    enable_controller_attach_detach: "{{ specification.enable_controller_attach_detach }}"
```

Comment:
As commented https://github.com/epiphany-platform/epiphany/issues/3190#issuecomment-1162894740, we don't want to setup this flag in kubeadm-join-node, so this argument shouldn't be present in this file.
Only location to configure this argument should be `ansible/playbooks/roles/kubernetes_master/templates/kubeadm-config.yml.j2`